grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy

```diff
@@ -61,6 +61,7 @@ class StockMovementApiController {
         stockMovement.requisitionStatusCodes = params.requisitionStatusCode ? params?.list("requisitionStatusCode") as RequisitionStatus[] : null
         stockMovement.requestType = params.requestType ? params.requestType as RequisitionType : null
         stockMovement.sourceType = params.sourceType ? params.sourceType as RequisitionSourceType : null
+        stockMovement.approvers = params.approver ? User.getAll(params.list("approver")) : null
```

Comment:
are we aware of the fact, that if somehow `User` of given id won't exist, it will fill the List with the `null` value? I assume Hibernate will ignore it when `createCriteria` is triggered, but just thought that if it doesn't, we can always just `findAll` by the identity (`it`), and it should filter those out

Reply:
you're right